editor/edit-post/header/plugins/index.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import { map } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { getSidebars, activateSidebar } from '../../../api/sidebar';
+import { MenuItemsGroup } from '../../../../components';
+import { getActivePanel, isEditorSidebarOpened } from '../../../store/selectors';
+import { connect } from 'react-redux';
+import { toggleSidebar } from '../../../store/actions';
+
+/**
+ * Renders a list of plugins that will activate different UI elements.
+ *
+ * @param {Object}   props Props.
+ * @param {Function} props.onSwitch        Function to call when a plugin is
+ *                                         switched to.
+ * @param {string}   props.activePanel     The currently active panel.
```

Comment:
Even though the JSDoc documentation differentiates with lower and uppercase differences, and I'd previously made an attempt to respect them, I think we ought to just standardize on always uppercasing, to avoid needing to consistently consult with the docs (or more likely, do it wrong).

Reply:
This could be a good discussion to have in #core-js. Tentatively I would say that keeping primitives lower case has value for a parser. This is consistent with PHP, where primitives are always lowercase and objects are always spelled the way they are defined in the `class` statement.

Doesn't block this PR, I think.